<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCrea</class>
 <widget class="QWidget" name="FormCrea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Caracteristiques</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="3" column="1">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Abilitées</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Modificateur</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>FOR</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Mod. autre</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>CON</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>DEX</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Mod. de race</string>
           </property>
          </widget>
         </item>
         <item row="6" column="4">
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="6" column="5">
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="4">
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QSpinBox" name="spinBox_3"/>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QSpinBox" name="spinBox_2"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>CHA</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QSpinBox" name="spinBox_4"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>SAG</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QSpinBox" name="spinBox_6"/>
         </item>
         <item row="6" column="3">
          <widget class="QSpinBox" name="spinBox_5"/>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="5">
          <widget class="QLabel" name="label_34">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QLabel" name="label_31">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Score</string>
           </property>
          </widget>
         </item>
         <item row="5" column="5">
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_35">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="5">
          <widget class="QLabel" name="label_32">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="7" column="4">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="7" column="5">
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>INT</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Score final</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="label_36">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="spinBox"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Compétences</string>
      </attribute>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>531</width>
         <height>231</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>260</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>260</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Dons</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Traits</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FormCrea</receiver>
   <slot>modifFor(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FormCrea</receiver>
   <slot>modifDex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FormCrea</receiver>
   <slot>modifCon(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FormCrea</receiver>
   <slot>modifInt(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FormCrea</receiver>
   <slot>modifSag(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FormCrea</receiver>
   <slot>modifCha(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FormCrea</receiver>
   <slot>modifModFor(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FormCrea</receiver>
   <slot>modifModDex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FormCrea</receiver>
   <slot>modifModCon(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FormCrea</receiver>
   <slot>modifModInt(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FormCrea</receiver>
   <slot>modifModSag(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FormCrea</receiver>
   <slot>modifModCha(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>modifFor(int)</slot>
  <slot>modifDex(int)</slot>
  <slot>modifCon(int)</slot>
  <slot>modifInt(int)</slot>
  <slot>modifSag(int)</slot>
  <slot>modifCha(int)</slot>
  <slot>modifModFor(int)</slot>
  <slot>modifModDex(int)</slot>
  <slot>modifModCon(int)</slot>
  <slot>modifModInt(int)</slot>
  <slot>modifModSag(int)</slot>
  <slot>modifModCha(int)</slot>
 </slots>
</ui>
